publication-commons/src/main/java/no/unit/nva/publication/model/business/TicketStatus.java

```diff
@@ -17,6 +18,7 @@
 public enum TicketStatus {
     PENDING(PENDING_STATUS, "REQUESTED"),
     COMPLETED(COMPLETED_STATUS, "APPROVED"),
+    NOT_RELEVANT(NOT_RELEVANT_STATUS, "NOT RELEVANT"),
```

Comment:
Horrible name, any better suggestion?

Reply:
What about Unpublished as PublicationStatus.UNPUBLISHED? 

Than it makes sense -> unpublishing publication follows to unpublishing of tickets.

Reply:
UNPUBLISHED status suggest that there are tickets that have a PUBLISHED status.

Reply:
Thats true, ask Rurik :)